common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -441,6 +441,6 @@ public long getLastSeenNotificationTime() {
     }
 
     public void setPrefsLastSeenNotificationTime(long lastSeenNotificationTime) {
-        prefs.edit().putLong(PREFS_LAST_SEEN_NOTIFICATION_TIME, lastSeenNotificationTime).commit();
+        prefs.edit().putLong(PREFS_LAST_SEEN_NOTIFICATION_TIME, lastSeenNotificationTime).apply();
```

Comment:
Based on the documentation of `apply()` method and lack of issues I suggest to use it over `commit()`

> You don't need to worry about Android component lifecycles and their interaction with apply() writing to disk. The framework makes sure in-flight disk writes from apply() complete before switching states.

source: https://developer.android.com/reference/android/content/SharedPreferences.Editor#apply()